async_remote.md

```diff
@@ -0,0 +1,27 @@
+[other sections soon to follow]
+
+# Knowledge transfer
+
+## Wiki checklist
+
+### Index
+  List and describe all the tools, services and things used in the project.
```

Comment:
I think it would benefit from being slightly _more_ specific what a tool or service is. 

Do you have in mind tools & services like external integrations (external APIs, Firebase, Maps SDK) and/or tooling (CodeClimate, Sentry, Grafana)? Or tools that power the app, like Python, Jetpack, RxKotlin, Flask, Hypothesis, Espresso?

Reply:
I would think external, since internal tools (i.e. technology stack) should be considered part of the architecture and hence describe somewhere else.